aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -257,17 +263,35 @@ When using the out-of-process hosting model, the app might not shut down immedia
 ::: moniker-end
 
 ## Start-up error page
-
 ::: moniker range=">= aspnetcore-2.2"
 
-*Only applies to out-of-process hosting.*
+Both inprocess and outofprocess have custom error pages when they fail to start the application.
+
+If the ASP.NET Core module fails to find either the inprocess or out of process request handler, a *500.0 In-Process/Out-Of-Process Handler Load Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-inproc.png)
```

Comment:
No images unless absolutely necessary. We're trying to purge as many as we can from the docs because we can't maintain them. If you can say in the text what the output produces, it's best to leave it to text.